apps/lacework-accounts/aws_eks_audit_log.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Quick note on location, I'm wondering if this should go under the util plan structure? https://github.com/lacework-dev/detc-resources/tree/main/plans/util